Add submission.yaml for example submission metadata (#160)
Kattis/problem-package-format: spec/2023-07-draft.md

| run_time_error                                    | Crashes for some test case. May output wrong answers or be too slow for other test cases.                          | Very rarely useful.

Every file or directory in these directories represents a separate solution.
It is mandatory to provide at least one accepted solution.

> Metadata abut the example submissions are provided in a UTF-8 encoded YAML file named `submissions.yaml` placed directly in the `submissions/` directory.

Reviewer: ```suggestion
Metadata about the example submissions are provided in a UTF-8 encoded YAML file named `submissions.yaml` placed directly in the `submissions/` directory.
```

> The top level keys in `submissions.yaml` are. globs matching example submissions.

Reviewer: ```suggestion
The top level keys in `submissions.yaml` are globs matching example submissions.
```

> E.g. `accepted/*` would match any submission in the `submissions/accepted/` directory.

Reviewer: ```suggestion
For example, `accepted/*` would match any submission in the `submissions/accepted/` directory.
```
